tests/tracking/fluent/test_fluent.py

```diff
@@ -1257,7 +1257,7 @@ def test_log_input(tmp_path):
     df.to_csv(path)
     dataset = from_pandas(df, source=path)
     with start_run() as run:
-        mlflow.log_input(dataset, "train")
+        mlflow.log_input(dataset, "train", {"foo": "baz"})
```

Comment:
Can we also make another call to `log_input` without the tags to ensure coverage of both cases?